source/documentation/administration_guide/topics/Host_Power_Management_settings_explained.adoc

```diff
@@ -10,7 +10,7 @@ The *Power Management* settings table contains the information required on the *
 |===
 |Field Name |Description
 |*Enable Power Management* |Enables power management on the host. Select this check box to enable the rest of the fields in the *Power Management* tab.
-|*Kdump integration* |Prevents the host from fencing while performing a kernel crash dump, so that the crash dump is not interrupted. In {enterprise-linux} 7.1 and later, kdump is available by default. If kdump is available on the host, but its configuration is not valid (the kdump service cannot be started), enabling *Kdump integration* will cause the host (re)installation to fail. If this is the case, see xref:fence_kdump_Advanced_Configuration[fence_kdump Advanced Configuration].
+|*Kdump integration* |Prevents the host from fencing while performing a kernel crash dump, so that the crash dump is not interrupted. In {enterprise-linux} 7.1 and later, kdump is available by default. If kdump is available on the host, but its configuration is not valid (the kdump service cannot be started), enabling *Kdump integration* will cause the host (re)installation to fail. If this is the case, see xref:fence_kdump_Advanced_Configuration[fence_kdump Advanced Configuration]. Turning on this option will require to reinstall the host for applying the correct fencing kdump configuration, see xref:sect-Host_Resilience[Host Resilience]
```

Comment:
Are you saying that the host needs to be reinstalled only when kdump is enabled, so that the fencing setting will be applied? Not when it's disabled? If that is the case, I recommend you phrase it more directly:

"If you enable kdump, you must reinstall the host in order to apply the correct fencing setting." I don't think you need the Host Resilience link. It's the same as the "fence_kdump Advanced Configuration" link in the previous sentence.

Reply:
Done